lute/book/model.py

```diff
@@ -50,6 +50,13 @@ def load(self, book_id):
             raise ValueError(f"No book with id {book_id} found")
         return self._build_business_book(dbb)
 
+    def find_by_title(self, book_title):
```

Comment:
This should probably also include the book language as a parameter, e.g if I'm learning Spanish and French, and both have a book called "lesson_1".  Small point only :-)

Reply:
Done.